onnxruntime/core/providers/xnnpack/detail/node_support_checker.cc

```diff
@@ -17,6 +17,8 @@
 // each operator provides a helper to check if supported
 #include "core/providers/xnnpack/nn/conv.h"
 #include "core/providers/xnnpack/nn/max_pool.h"
+#include "core/providers/xnnpack/nn/gemm.h"
```

Comment:
gemm and matmul should be under path `xnnpack/math/` instead of `xnnpack/nn`. see other folders like `cpu` or `cuda` under `providers` folder.

Reply:
Updated in following commits now the new ops are under xnnpack/math